_maps/map_files/Mafia/mafia_spiderclan.dmm

```diff
@@ -91,13 +91,7 @@
 /turf/open/floor/plating,
 /area/centcom/mafia)
 "B" = (
-/obj/structure/showcase{
-	density = 0;
-	desc = "Welp, only one way to recover your honour.";
-	icon = 'icons/obj/items_and_weapons.dmi';
-	icon_state = "katana";
-	name = "seppuku katana"
-	},
+/obj/structure/showcase,
```

Comment:
Shouldn't this be `/obj/structure/showcase/katana` instead of the base showcase? Looks good otherwise.

Reply:
Thanks for catching that